src/main/webapp/WEB-INF/jsp/expiredPasswordResetLink.jsp

```diff
@@ -15,10 +15,6 @@
         <p class="lede">
             <spring:message code="public.user.password.reset.expired.text"/>
         </p>
-        <p>
```

Comment:
Note that this will be added back (thought slightly changed) in a follow-up ticket.